Logger/LogFactory.cs

```diff
@@ -2,10 +2,20 @@
 {
     public class LogFactory
     {
+        private string FilePath;
```

Comment:
Fields should be underscore PascalCase
```suggestion
        private string _FilePath;
```